src/test/java/it/pagopa/swclient/mil/controller/model/CommonHeaderTest.java

```diff
@@ -0,0 +1,34 @@
+package it.pagopa.swclient.mil.controller.model;
+
+import it.pagopa.swclient.mil.util.TestBase;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class CommonHeaderTest extends TestBase {
```

Comment:
please do not use inheritance just to get a validator. Maybe use static utility or configurationTest to inject the bean (possible in spring, don't know in Quarkus), e.g.
```
static final Validator validator() {
    return Validation.buildDefaultValidatorFactory().getValidator();
}
```

Reply:
Resolved in [https://github.com/pagopa/mil-terminal-registry/pull/25/commits/8c55df2932810878f744b7992b7a535c9c819587](https://github.com/pagopa/mil-terminal-registry/pull/url)